components/service/glean/build.gradle

```diff
@@ -14,8 +14,8 @@ apply plugin: 'kotlin-android'
  * created during unit testing.
  * This uses a specific version of the schema identified by a git commit hash.
  */
-String GLEAN_PING_SCHEMA_GIT_HASH = "8939bcb"
-String GLEAN_PING_SCHEMA_URL = "https://raw.githubusercontent.com/mozilla-services/mozilla-pipeline-schemas/$GLEAN_PING_SCHEMA_GIT_HASH/schemas/glean/baseline/baseline.1.schema.json"
+String GLEAN_PING_SCHEMA_GIT_HASH = "a2c1ee1"
+String GLEAN_PING_SCHEMA_URL = "https://raw.githubusercontent.com/mdboom/mozilla-pipeline-schemas/$GLEAN_PING_SCHEMA_GIT_HASH/schemas/glean/baseline/baseline.1.schema.json"
```

Comment:
This is updating the schemas we're checking against to use the work-in-progress in my personal fork [this PR](https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/254).  We will need to change this back to the mainline fork and a new git commit before merging this.